README.md

```diff
@@ -126,19 +128,26 @@ jobs:
 
 ## Creating an API Key
 
-To create an API key to use with this action go to [Account Settings > API Keys](https://app.overmind.tech/changes?settings=1&activeTab=api-keys) and click "New API Key".
+To create an API key to use with this action go to [Account Settings > API Keys](https://app.overmind.tech/settings/api-keys) and click "New API Key".
 
 ![api keys auth window](./doc/api_keys.png)
 
-Give the key a name e.g. "Github Actions" and select the `changes:write` permission and click "Confirm". This will create the API key and authorize it. The key should then display as "Ready" in the UI.
+Give the key a name e.g. "Github Actions" and select the `account:read`, `changes:write`, `config:write`, `request:receive`, and `source:write` permissions and click "Confirm". This will create the API key and authorize it. The key should then display as "Ready" in the UI.
 
 You can then copy the API key and [create a secret](https://docs.github.com/en/actions/security-guides/using-secrets-in-github-actions#creating-secrets-for-a-repository) called `OVM_API_KEY` in Github Actions. The action will now be ready to use.
 
+## Enterprise support
+
+For Enterprise customers, `submit-plan`, `start-change` and `end-change` actions support an `app:` key in the `with` section of the action.
```

Comment:
```suggestion
For Enterprise customers, `submit-plan`, `start-change` and `end-change` actions support an `app:` key in the `with` section of the action which allows you to target an on-prem instance of Overmind e.g.

      - uses: overmindtech/actions/submit-plan@main
        id: submit-plan
        with:
          ovm-api-key: ${{ secrets.OVM_API_KEY }}
          plan-json: ./tfplan.json
          plan-output: ./terraform_log
          app: https://mycompany.overmind.tech
```

Not that I can't actually put the backticks around that YAML in a suggestion so it'll need to be added after the fact